b2share/modules/deposit/permissions.py

```diff
@@ -32,22 +32,18 @@
 from jsonpatch import apply_patch
 from flask_principal import UserNeed
 from invenio_access.permissions import (
-    superuser_access, ParameterizedActionNeed, DynamicPermission
```

Comment:
TODO: change back to DynamicPermission 

https://github.com/EUDAT-B2SHARE/b2share/pull/1842/files#r594444350